src/block_cache/block_io.c

```diff
@@ -376,7 +376,7 @@ __wt_blkcache_read_multi(WT_SESSION_IMPL *session, WT_ITEM **buf, size_t *buf_co
     if (bm->read_multiple == NULL) {
         WT_RET(__wt_calloc_def(session, 1, &tmp));
         WT_CLEAR(tmp[0]);
-        WT_ERR(__wt_blkcache_read(session, &tmp[0], block_meta, addr, addr_size));
+        WT_ERR(__wt_blkcache_read(session, &tmp[0], NULL, addr, addr_size));
```

Comment:
What was the motivation for this?

Reply:
I found we read garbage data for block meta in this code path and causing us to hit assertions later.

Reply:
Thanks